### PR TITLE
upgrade: Remove CEPH related code

### PR DESCRIPTION
**Pull request by skazi0**
Crowbar managed CEPH clusters are not supported in Cloud7->8 upgrade.

### CODE REVIEW

File: assets/app/features/upgrade/upgrade.constants.js [hunk 1]
```diff
@@ -3,8 +3,7 @@
     angular
         .module('crowbarApp.upgrade')
         .constant('ADDONS_PRECHECK_MAP', {
-            'ha': ['clusters_healthy'],
-            'ceph': ['ceph_healthy']
```
Comment (jsuchome) [inline assets/app/features/upgrade/upgrade.constants.js:7]: Note there there's this new group called `ceph_status` (which could contain crowbar_ceph_present error)

Comment (skazi0) [reply assets/app/features/upgrade/upgrade.constants.js:7]: yup, updating the PR right now.

**Comment by jsuchome** [comment]
@skazi0 what's the branching and packaging policy here? AFAIK we need this bit on SOC7 side (but not to mess with SOC7 side of 6-7 upgrade)

**Comment by skazi0** [comment]
@jsuchome re branching policy... this might get tricky. we have normal branching so far so Cloud7 has the Cloud6->7 version. I'll have to think about this...

File: assets/app/features/upgrade/upgrade.constants.js (hunk 1 above)
Comment (skazi0) [reply assets/app/features/upgrade/upgrade.constants.js:7]: should be fixed now. see my assumptions comment below.

**Comment by skazi0** [comment]
assumptions:
- no ceph addon at all (i.e. not reported from backend or ignored)
- no ceph repochecks
- no ceph health check (negative ceph presence check instead)
- anything more?

**Comment by jgravel** [comment]
There's a reference of 'ceph_healthy' at https://github.com/crowbar/crowbar-ui/blob/master/routes/api/upgrade/prechecks.js#L20. Should that line be updated as well?

**Review by jsuchome — APPROVED** [review]

**Comment by skazi0** [comment]
rebased and ready for final review

**Review by jsuchome — APPROVED** [review]

**Review by jgravel — APPROVED** [review]

**Pull request merged**